examples/kfdef.yaml

```diff
@@ -34,7 +34,11 @@ spec:
         path: istio/cluster-local-gateway
     name: cluster-local-gateway
   - kustomizeConfig:
+      overlays:
+      - https-gateway
```

Comment:
I am just curious, why do we need this overlay? I didn't find any docs about this, can you please point/explain to me a bit?

Reply:
by default, kubeflow exposed with HTTP but it's not secure and we decided to expose kubeflow using HTTPS, that's why we need that

Reply:
Can I find any official documentation which is actually using this?

Reply:
it's just an overlay that you can add if you want to have https, that's it